Added Contributing.md

nimit2801 opened the pull request:
- Added the contributing.md for [Hackathons](https://github.com/hackclub/hackathons)
- With some cool gifs and informative images
- Solving #37 
- Please review thank you.

zanedb commented on src/components/EventCard.js:
```diff
@@ -137,6 +132,7 @@ export default ({
     <EventCard bg={banner}>
       <AssociatedSeal mlhAssociated={mlh} />
       <LogoContainer>
+        {console.log(id, logo, banner, website, name)}
```
Yup, this was temporary :)

zanedb commented on src/components/EventCard.js:
```diff
@@ -49,37 +49,32 @@ const EventCard = Flex.withComponent(Tilt).extend.attrs({
-      alt="MLH logo"
-      mlhAssociated={props.mlhAssociated}
-    />
+    <>
```
Would like to, but unfortunately it throws a ESLint error. see: https://stackoverflow.com/a/9534132

MaxWofford commented:
Closing due to inactivity. Feel free to reopen if you think it should be

nimit2801 committed:
Contrinuting.md updated

nimit2801 committed:
Contrinuting.md updated v2

nimit2801 committed:
Contrinuting.md updated v3

sampoder reviewed (changes requested):
Thanks for contributing :)

Unfortunately, I don't think this is up to standard. A few pointers:

- Headings should only be used for headings not body text
- The grammar needs significant improving 
- The "vibe" of the text feels off, it feels unprofessional but not in a good way

Hope you can understand

nimit2801 committed:
Update CONTRIBUTING.md

Update CONTRIBUTING.md
1.  Grammatical errors
2.  Made the CONTRIBUTING.md more professional
3. Deleted the unwanted use headings in the body

nimit2801 commented:
@sampoder I've made the changes requested.
Thank you for the review.

nimit2801 committed:
Updated Contributing.md

Deleted all unnecessary headers.

sampoder reviewed (changes requested):
It's better, but there are still a lot of grammar errors

nimit2801 commented:
> 
> 
> It's better, but there are still a lot of grammar errors

Can you tell me which lines I should, improve or some suggestions.
Thank You.

sampoder commented on CONTRIBUTING.md:
```diff
@@ -1,26 +1,26 @@
+# Contribution Guidelines
 
-#### This document contains a set of guidelines which any contributions to any Hack Club repository must follow.
+#### This document contains a set of guidelines that any contributions to any Hack Club repository must follow.
```
G

sampoder commented on CONTRIBUTING.md:
```diff
@@ -1,26 +1,26 @@
 ### Step 1: Fork repo.
 
-#### Fork the original repo [hackathons](https://github.com/hackclub/hackathons) !
+Fork the original repo [hackathons](https://github.com/hackclub/hackathons) !
```
G

sampoder commented on CONTRIBUTING.md:
```diff
@@ -1,26 +1,26 @@
 ![forked](https://cloud-99tamsbv5-hack-club-bot.vercel.app/1forked.png)
 
-#### Something in your URL would look like:
+Something in your URL would look like this:
```
G

sampoder commented on CONTRIBUTING.md:
```diff
@@ -38,9 +38,9 @@ cd hackathons
+### Step 4: Make your Awesome changes!
 
-![Awsome changes](https://cloud-qoynsmfj3-hack-club-bot.vercel.app/0tenor.gif)
+eg: In my case, I was adding the CONTRIBUTING.md
```
G

sampoder commented:
View above comments. Each has a grammar / phrasing issue.

sampoder commented:
Also this document still doesn't feel right. IMO it should be made more professional

nimit2801 commented:
I'll reformat it, and update my PR!
@sampoder thanks for the review.

nimit2801 committed:
Merge branch 'hackclub:main' into CONTRIBUTING.md
